### PR TITLE
fix(client): prevent code duplication when using help button

### LINKED ISSUE #46153: "Ask for Help" button duplicates the code in forum post
### Describe the Issue

The "Ask for Help" button duplicates the HTML and CSS code when ported over to the forum.
This issue happens with steps/projects with multiple files. (e.g. index.html and styles.css)

### Affected Page

Any steps/projects with multiple files.

### Steps to Reproduce

1. Go to any step/project with an editor with mutiple files. [For example this one](https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-responsive-web-design-by-building-a-piano/step-2) 
2. Try submitting incorrect code until Ask for Help button appears
3. Click Ask for Help and create a post on the forum
4. See the code in the draft post

### Expected behavior

The code should not be duplicated in the forum post

### Screenshots

image

### System

- Device: Laptop
- OS: Windows 10
- Browser: Chrome
- Version: 101.0.4951.67

### Additional context

I noticed it seeing the forum posts below and I could reproduce the issue myself.
https://forum.freecodecamp.org/t/step-49-cant-figure-this-one-out-its-so-simple-though/511630
https://forum.freecodecamp.org/t/why-the-image-is-not-seting-on-html-page/511638

### PR DESCRIPTION
**Pull request by Nirajn2311**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46153 

I wasn't sure if `challengeFile.history` would always contain the file name in the first index so I went with `name` and `ext`

### CODE REVIEW

File: client/src/templates/Challenges/redux/create-question-epic.js [hunk 1]
```diff
@@ -21,10 +21,10 @@ function filesToMarkdown(challengeFiles = {}) {
       return fileString;
     }
     const fileName = moreThanOneFile
-      ? `\\ file: ${challengeFile.contents}`
+      ? `\\ file: ${challengeFile.name}.${challengeFile.ext}\n`
```
Comment (ShaunSHamilton) [inline client/src/templates/Challenges/redux/create-question-epic.js:24]:
```suggestion
      ? `\* file: ${challengeFile.name}.${challengeFile.ext} */\n`
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
This suggestion is not entirely related to the issue, but I think makes more sense considering `\` is not a valid comment style for anything.

Unless I am misunderstanding why we did that...

File: client/src/templates/Challenges/redux/create-question-epic.js (hunk 1 above)
Comment (Nirajn2311) [reply client/src/templates/Challenges/redux/create-question-epic.js:24]: Yea this is a lot better. If anyone copies the code then they won't face any errors.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by raisedadead — APPROVED** [review]

**Pull request merged**